components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1223 @@
+$$
+ {\bf D}^u_v=\frac{\partial }{\partial z}
+\left( \nu_v \frac{\partial {\bf u}}{\partial z} \right)
+$$ (continuous-v_mom_diff-R13App)
+$$
+ {\bf D}^u_h=\nu_h \nabla^2 \bf u = \nu_h(\nabla \delta + {\bf k}\times \nabla \eta)
+$$ (continuous-h_mom_diff-R13App)
+and may also include a del-4 term, Rayleigh drag, and bottom drag. Momentum forcing is due to wind stress.
+
+Tracer diffusion terms include
+
+$$
+D^\varphi =  D^\varphi_h + D^\varphi_v
+$$ (continuous-tracer_diff-R13App)
+$$
+ D^\varphi_v = \rho \frac{\partial }{\partial z}
+  \left( \kappa_v \frac{\partial \varphi}{\partial z} \right),
+$$ (continuous-v_tr_diff-R13App)
+$$
+ D^\varphi_h = \nabla\cdot\left(\rho \kappa_h \nabla\varphi \right).
+$$ (continuous-h_tr_diff-R13App)
+and may also include a del-4 term. Sources and sinks include surface fluxes from the atmosphere and land, and bio-geo-chemical reactions.
+
+<!---
+$$
+\frac{\partial \mathbf{u}}{\partial t}
++ q\left(h\mathbf{u}^{\perp}\right) 
+= 
+- \frac{1}{\rho} \nabla p
+- \nabla K
++ \nu_2 \nabla_r^2 \mathbf{u}
+- \nu_4 \nabla_r^4 \mathbf{u}
++ \frac{\partial }{\partial z} \left( \nu_v \frac{\partial {\mathbf u}}{\partial z} \right)
++ \mathcal{D} 
++ \mathcal{F}
+$$ (continuous-momentum)
+
+$$
+\frac{\partial h}{\partial t} + \nabla_r \cdot \left(h \mathbf{u}\right) + \omega_{bot} - \omega_{top}= 0,
+$$ (continuous-mass)
+
+$$
+\frac{\partial h \varphi}{\partial t} + \nabla_r \cdot \left(h \mathbf{u} \varphi\right) 
++ \varphi\omega_{bot} - \varphi\omega_{top} 
+= 
+\kappa_2 h \nabla_r^2 \varphi 
+- \kappa_4 h \nabla_r^4 \varphi
++ h \frac{\partial }{\partial z} \left( \kappa_v \frac{\partial \varphi}{\partial z} \right).
+$$ (continuous-tracer)
+--->
+
+### Layered Equations
+
+Omega-1 will be non-Boussinesq, which means that density is not assumed to be constant in any terms. This results in a mass conservation equation, rather than a volume conservation equation. The fully continuous equations were presented in the previous section. Here we derive the layered equations by discretizing in the vertical, while the horizontal remains continuous. We discretize by integrating in the vertical from the surface $z=s_k^{bot}$ to $z=s_k^{top}$ for the layer with index $k$, as described in [Ringler et al. 2013](https://www.sciencedirect.com/science/article/pii/S1463500313000760) Appendix A.2. Equivalently, we can vertically integrate from a lower pressure surface $p=\sigma_k^{bot}$ to $p=\sigma_k^{top}$ where $p$ and $z$ are related by the hydrostatic pressure equation [](#continuous-hydrostatic-pressure).
+
+For non-Boussinesq layered equations we begin by defining the pressure-thickness of layer $k$ as
+
+$$ (sigma-int-to-surf)
+
+where $\eta$ is the sea surface height and $p^{surf}$ is the surface pressure. Note that the negative sign means that pressure increases with depth, so positive $p$ points downward, so that the $top$ and $bot$ extents of the integration limits are flipped in [](#depth-pressure-integral-conversion).
```

Comment:
The link here doesn't seem to be working.

Reply:
Done. I missed a copy-paste from another doc.